app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine is outdated")
+
+        if not visitor.get("wearing_a_mask", False):
```

Comment:
You are using the `get` method to check if the visitor is wearing a mask. This is fine, but it would be more pythonic to use the `in` keyword to check if the `wearing_a_mask` key is in the `visitor` dictionary. For example: `if 'wearing_a_mask' not in visitor or not visitor['wearing_a_mask']:`. This way, you are explicitly checking if the key is present in the dictionary and if its value is `True`.